bulk_deploy/tsuru_api_service.rb

```diff
@@ -0,0 +1,119 @@
+require_relative './../dsl/ssh_helper'
+
+class TsuruAPIService
+
+  def initialize(logger:, api_client:, tsuru_home:)
+    @logger = logger
+    @api_client = api_client
+    @tsuru_home = tsuru_home
+  end
+
+  def api_client
```

Comment:
For these kind of constructions, Ruby provides the helpers [attr_accessor](http://ruby-doc.org/core-2.1.0/Module.html#method-i-attr_accessor), [attr_reader](http://ruby-doc.org/core-2.1.0/Module.html#method-i-attr_reader) and [attr_writter](http://ruby-doc.org/core-2.1.0/Module.html#method-i-attr_writer)

You can add the line:

```
attr_reader :api_client, :logger
```